eng/imports/HostAgnostic.props

```diff
@@ -38,6 +38,9 @@
     <!-- VS MEF -->
     <PackageReference Include="Microsoft.VisualStudio.Composition" />
 
+    <!-- Pin MessagePack to a patched version with security vulnerable fix -->
+    <PackageReference Include="MessagePack" />
```

Comment:
If we add this here, we should remove it from `VisualStudio.props` (which imports this file) so that we don't include it twice in projects.